src/mono/wasm/runtime/loader/blazor/WebAssemblyResourceLoader.ts

```diff
@@ -127,10 +127,16 @@ export class WebAssemblyResourceLoader {
         // Note that if cacheBootResources was explicitly disabled, we also bypass hash checking
         // This is to give developers an easy opt-out from the entire caching/validation flow if
         // there's anything they don't like about it.
-        return fetch(url, {
+        const fetchOptions: RequestInit = {
             cache: networkFetchCacheMode,
             integrity: this.bootConfig.cacheBootResources ? contentHash : undefined,
-        });
+        };
+
+        if (resourceType === "configuration") {
```

Comment:
please add some comment explaining this

Reply:
It's migrated behavior from Blazor https://github.com/dotnet/aspnetcore/pull/19544.
I didn't found any special comment about it

Reply:
Does it mean that credentials are sent in order to allow server to provide user specific file ? If so, it would deserve such comment.